azurerm/internal/services/containers/kubernetes_cluster_data_source.go

```diff
@@ -118,6 +118,19 @@ func dataSourceKubernetesCluster() *schema.Resource {
 								},
 							},
 						},
+
+						"open_service_mesh": {
```

Comment:
same for the data source - can we add this to the `addon_profile` block: https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/kubernetes_cluster.html.markdown#L91-L101

and the specific fields below that as here: https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/kubernetes_cluster.html.markdown#L255-L257

Reply:
@tombuildsstuff Added, thanks